pkg/watch/key_watcher_windows.go

```diff
@@ -0,0 +1,20 @@
+package watch
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+// makeRaw is forked from golang.org/x/term
```

Comment:
What is the `makeRaw` that you refer to here? I don't find it anywhere except this comment:
```shell
$  grep -r makeRaw pkg/      
pkg/watch/key_watcher_windows.go:// makeRaw is forked from golang.org/x/term

```

Reply:
`enableCharInput` just below. Sorry I renamed the function name, but didn't update the comment